src/control/README.md

```diff
@@ -44,4 +44,4 @@ Please refer to package-specific README's.
 
 ## User Documentation
 
-- [online documentation](https://daos-stack.github.io/)
```

Comment:
Perhaps the section title could be changed to "User Documentation for the Latest release" since the /latest URL refers to 2.0 documentation (currently)?

Or would it make sense (or just be more confusing) to provide 2 links, one that refers to https://docs.daos.io/latest (for the lastest release) and another that refers to https://docs.daos.io/master that reflects the latest pre-released documentation (currently "v2.3" in advance of the v2.4 release)? Of course that might need to be done in many places, which may not be worthwhile.